src/libopensc/card-cardos.c

```diff
@@ -839,12 +1029,12 @@ cardos_set_security_env(sc_card_t *card,
 
 				sc_log(card->ctx, "is signature");
 				sc_log(card->ctx, "Adding ID %d at index %d", algorithm_id, algorithm_id_count);
-				algorithm_ids_in_tokeninfo[algorithm_id_count++] = algorithm_id;
+				priv->algorithm_ids_in_tokeninfo[algorithm_id_count++] = algorithm_id;
 			}
 			sc_log(card->ctx, "reference=%d, mechanism=%d, operations=%d, algo_ref=%d",
 					alg.reference, alg.mechanism, alg.operations, alg.algo_ref);
 		}
-		algorithm_ids_in_tokeninfo_count = algorithm_id_count;
+		priv -> algorithm_ids_in_tokeninfo_count = algorithm_id_count;
```

Comment:
clang-format suggested the following change:


```
-		priv -> algorithm_ids_in_tokeninfo_count = algorithm_id_count;
+		priv->algorithm_ids_in_tokeninfo_count = algorithm_id_count;
```

<!-- identifier: clang-format -->
<!-- File: src/libopensc/card-cardos.c -->